components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,660 @@
+
+The perpendicular gradient maps a scalar at vertices to an edge-tangential vector component,
+$$
+\left(\nabla \omega_v\right)_e^\perp = \frac{1}{l_e} \sum_{v\in VE(e)} -t_{e,v}\omega_v.
+$$
+Like the standard gradient, in practice we drop the sign indicator $t_{e,v}$ and rewrite it using the index ordering,
+$$
+\left(\nabla \omega_v\right)_e^\perp = \frac{\omega_{v2} - \omega_{v1}}{l_e},
+$$
+where the positive vector ${\bf n}^\perp$ is 90$^o$ to the left of ${\bf n}$. The indices are ordered such that ${\bf n}^\perp$ points from $v_1$ to $v_2$, which corresponds to VerticesOnEdge(0,iEdge) and VerticesOnEdge(1,iEdge) in the code.
+
+
+#### 3.2.6. Cell to Edge Interpolation
+This is simply a mid-point averaging from cell centers to the adjoining edge.
+
+$$
+[h_i]_e = \frac{1}{2} \sum_{i\in CE(e)} h_i
+$$
+
+#### 3.2.7. Vertex to Edge Interpolation
+This is a mid-point averaging from vertices to the middle of the connecting edge.
+
+$$
+[\omega_v]_e = \frac{1}{2} \sum_{v\in VE(e)} \omega_v
+$$
+
+#### 3.2.8. Cell to Vertex Interpolation
+This is an area-weighted average at a vertex from the three surrounding cells. Here ${\tilde A}_v$ is the dual triangle area centered on the vertex, and ${\hat A}_{i,v}$ is the kite area of cell $i$ with the kite point at vertex $v$.
+
+$$
+[h_i]_v = \frac{1}{{\hat A}_{i,v}} \sum_{i\in CV(v)} h_i {\tilde A}_v
+$$
+
+#### 3.2.9. Vector from Edge to Cell
+
+The prognostic velocity variable on the edge is the edge normal velocity, $u_e$. The tangential velocity $u_e^\perp$ is computed diagnostically from $u_e$. In addition, the full vector may be computed at the cell center from the edge normal velocities $u_e$ of the edges on that cell. That is done in MPAS with radial basis functions, and is explained in this [previous design document](https://github.com/MPAS-Dev/MPAS-Documents/blob/master/shared/rbf_design/rbf.pdf).
+
+### 3.3 Momentum Terms
+
+The computation of each term in (4-6) is now described in detail, along with alternative formulations.
+
+#### 3.3.1. Kinetic energy gradient
+
+The kinetic energy gradient term is the non-rotational part of the nonlinear advection. Fundamentally, it maps the prognostic edge-normal velocity from edges back to an edge scalar quantity. We use the standard gradient formulation from cell center to edge,
+
+$$
+\nabla K_i = \frac{1}{d_e} \sum_{i\in CE(e)} -n_{e,i}K_i.
+$$
+
+There are several methods to compute the cell-centered kinetic energy. The first uses the edge-normal velocities as in
+[Ringler et al. 2010](https://www.sciencedirect.com/science/article/pii/S0021999109006780) equation 63,
+
+$$
+K_i = \frac{1}{A_i} \sum_{e\in EC(c)} \frac{{\bar A}_{e}}{4}u_e^2
```

Comment:
`A_e` is not defined and is missing from the variables table.

Reply:
done, thanks.